test/python/algorithms/test_qaoa.py

```diff
@@ -314,7 +314,8 @@ def test_qaoa_random_initial_point(self):
         qaoa = QAOA(optimizer=NELDER_MEAD(disp=True), reps=1, quantum_instance=q_i)
         _ = qaoa.compute_minimum_eigenvalue(operator=qubit_op)
 
-        np.testing.assert_almost_equal([-0.8901, -5.8885], qaoa.optimal_params, decimal=4)
+        # np.testing.assert_almost_equal([-0.8901, -5.8885], qaoa.optimal_params, decimal=4)
+        np.testing.assert_almost_equal([-0.2337, - 5.9668], qaoa.optimal_params, decimal=4)
```

Comment:
Do you have any idea why the solution changed here? It used to pass before the refactor. Has something changed around the ansatz that this is expected - such as slightly different parameter order now?

Reply:
This is due to recent changes in opflow (last few PRs, the error occurred on Jan 30th). Once I updated the branch I had a problem with this test. Realized that optimal parameters are different now.

Reply:
Well, maybe it is not that simple as it looks like. The same test in `optimization` looks good with the old numbers.

Reply:
Ok, the other tests pass so maybe this is fine - the code as far as I can see seems fine.